phase0/CRC.md

```diff
@@ -0,0 +1,205 @@
+---
+marp: true
+title: Marp CLI example
+description: Hosting Marp slide deck on the web
+theme: gaia
+---
+
+<style>
+:root {
+  font-size: 24px;
+}
+</style>
+
+<!-- _class: lead -->
+# Entity Classes
+
+---
+
+# `abstract class IndividualCharacter`
+
+### Responsibilities
+* Health: Attribute, int, denotes the total amount of health the individual has
+* Position: Attribute, array with x, y, where on the map the IC is currently located
```

Comment:
should be handled by collision/map/world system

`IndividualCharacter` will ofc have a position, but that attribute should come from extending some interface